application/src/main/java/io/redlink/smarti/properties/HttpCallbackProperties.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2017 Redlink GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package io.redlink.smarti.properties;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "http.callback")
+public class HttpCallbackProperties {
+
+    private int retryCount = 3;
+
+    private ProxyProperties proxy = new ProxyProperties();
+
+    public int getRetryCount() {
+        return retryCount;
+    }
+
+    public void setRetryCount(int retryCount) {
+        this.retryCount = retryCount;
+    }
+
+    public ProxyProperties getProxy() {
+        return proxy;
+    }
+
+    public void setProxy(ProxyProperties proxy) {
+        this.proxy = proxy;
+    }
+
+    public static class ProxyProperties {
```

Comment:
Don't we have already proxy configuration properties in the `application.properties` file?